server/index.js

```diff
@@ -90,6 +90,21 @@ app.get("/v2/query", async (req, res) => {
   res.send({ location, results });
 });
 
+app.get("/v2/queryByStoreId", async (req, res) => {
+  const { query } = req;
+  let params = {
+    storeId: query.storeId,
```

Comment:
since store id is mandatory here, we could potentially throw a 400 if it's missing